packages/core/src/services/documentLogs/computeDocumentLogsWithMetadata.test.ts

```diff
@@ -3,7 +3,7 @@ import { describe, expect, it } from 'vitest'
 import { mergeCommit } from '../../services/commits'
 import { updateDocument } from '../../services/documents'
 import * as factories from '../../tests/factories'
-import { computeDocumentLogsWithMetadata } from './computeDocumentLogsWithMetadata'
+import { computeDocumentLogsWithMetadataQuery } from './computeDocumentLogsWithMetadata'
```

Comment:
By convention if a query service return the lazy query call it `somethingQuery` so we know